.github/stability.md

```diff
@@ -0,0 +1,58 @@
+## Stability
+
+In short: `windows-rs` is still under active development, and none of the crates
+under the org are considered "stable". Some crates such as `windows-sys` might
+be closer than others, but until a crate reaches `1.0.0` it may be subject to
+change.
+
+## Release schedule and versioning
+
+We're set to release new versions of crates every 3 weeks on Tuesdays. Currently all crates
+are pre-1.0 and have their semver-minor version bumped on release. We are
+looking for ways to version and release crates individually. Once we do, we will
+update this section with our versioning strategy.
+
+## Metadata Stability
+
+All crates in `windows-rs` interact with [Windows Metadata (`.winmd`)
+files](https://github.com/microsoft/win32metadata) which describe the various
+windows APIs. All Windows APIs can be split into two categories:
+`win32` APIs and `WinRT/COM` APIs. The way we bind to these APIs is different,
+but in the metadata they also have different stability guarantees:
+
+| Metadata category | Stable? |
+|-------------------|---------|
+| `Win32`           | ❌       |
+| `WinRT/COM`       | ✅       |
```

Comment:
Is the WinRT stable in the semver sense or only stable in the "eh, it doesn't change that often". We might want to define what we mean by stable metadata.